Sandbox/ListPatterns.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Sandbox;
+
+public static class ListPatterns
+{
+    public static void Patterns()
```

Comment:
Where do you need that?

Reply:
This project acts as a test for your StyleCop configuration. It's not exhaustive, but it lists some possible scenarios. If we were to break the configuration (or StyleCop were to break it) we'd notice it via a build error in this project.